docs/faq.rst

```diff
@@ -487,6 +487,45 @@ We **strongly** advice against downgrading the TLS version and ciphers!
 
 *Issue reference:* `363`_, `698`_.
 
+Why does Compose complain about the yaml syntax
+```````````````````````````````````````````````
+
+In many cases, Docker Compose will complain about the yaml syntax because it is too old. It is especially true if you installed Docker Compose as part of your GNU/Linux distirbution package system.
```

Comment:
```suggestion
In many cases, Docker Compose will complain about the yaml syntax because it is too old. It is especially true if you installed Docker Compose as part of your GNU/Linux distribution package system.
```